app/app/api/src/main/kotlin/de/stustapay/api/apis/AuthApi.kt

```diff
@@ -0,0 +1,103 @@
+/**
+ *
+ * Please note:
+ * This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * Do not edit this file manually.
+ *
+ */
+
+@file:Suppress(
+    "ArrayInDataClass",
+    "EnumEntryName",
+    "RemoveRedundantQualifierName",
+    "UnusedImport"
+)
+
+package de.stustapay.api.apis
+
+import kotlinx.serialization.Contextual
+
+
+import de.stustapay.api.models.TerminalRegistrationPayload
+import de.stustapay.api.models.TerminalRegistrationSuccess
+
+import de.stustapay.api.infrastructure.*
+import io.ktor.client.HttpClientConfig
+import io.ktor.client.request.forms.formData
+import io.ktor.client.engine.HttpClientEngine
+import io.ktor.http.ParametersBuilder
+
+    open class AuthApi(
+    baseUrl: String = ApiClient.BASE_URL,
+    httpClientEngine: HttpClientEngine? = null,
+    httpClientConfig: ((HttpClientConfig<*>) -> Unit)? = null,
+    ) : ApiClient(
+        baseUrl,
+        httpClientEngine,
+        httpClientConfig,
+    ) {
+
+        /**
+        * Log out this Terminal
+        * 
+         * @return void
+        */
+        open suspend fun logoutTerminal(): HttpResponse<Unit> {
+
+            val localVariableAuthNames = listOf<String>("OAuth2PasswordBearer")
+
+            val localVariableBody = 
+                    io.ktor.client.utils.EmptyContent
+
+            val localVariableQuery = mutableMapOf<String, List<String>>()
+
+            val localVariableHeaders = mutableMapOf<String, String>()
+
+            val localVariableConfig = RequestConfig<kotlin.Any?>(
+            RequestMethod.POST,
+            "/auth/logout_terminal",
+            query = localVariableQuery,
+            headers = localVariableHeaders,
+            requiresAuthentication = true,
+            )
+
+            return request(
+            localVariableConfig,
+            localVariableBody,
+            localVariableAuthNames
+            ).wrap()
```

Comment:
Yeah, this code is generated by the OpenAPI generator and I wasn't able to find a way to adjust the formatting using the provided config parameters. While it would probably be possible to run the code through an external formatting tool, that would add additional dependencies, which I would rather avoid (unless there's a good solution that integrates well into the gradle pipeline). However, since this code shouldn't ever have to be edited manually, I think it's acceptable to keep the broken formatting (or fix the code generation upstream ^^).